lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
:D I understand. I can't focus on the laptop screen without reading glasses
anymore (who'd have thought that 51 is old) and have used multiple screens
since the 90s. I find that the modern 32" flat screen HDTVs are awesome
mounted on the wall at about 4 ft - cheapest and best monitors I ever
bought. And a side-by-side view of two files in atom fits perfect at about
100 characters though I have learned to habitually toggle the directory
view in and out. :D

On Wed, Jul 27, 2016 at 1:07 PM, Joe Lencioni <notifications@github.com>
wrote:

> In lib/ImportStatements.js
> <https://github.com/Galooshi/import-js/pull/321#discussion_r72480273>:
>
> > @@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
> >  import Configuration from './Configuration';
>
> I'm okay with 100 I guess--that's what we use at work. I've been resisting
> because I usually work in a terminal split where 80 characters is just
> right, but perhaps I just need to get with the times...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Galooshi/import-js/pull/321/files/5138c74aaed901105f9957cf22129203e9b5de70..7b1c648f01761d447fe1b5193caffe85d5d72636#r72480273>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH5TYhAsMBVdaYXzlV3vUDfblZ0O34A9ks5qZ5BBgaJpZM4JU6-_>
> .
>